src/lib/Commands/TestCommand/Session.js

```diff
@@ -213,6 +213,15 @@ class Session extends EventEmitter {
 
       case 'DEVICE_DISCONNECTED':
 
+        if (this.allowDisconnect) {
+          this.emit('message', {
+          type: 'info',
```

Comment:
there is something wrong with alignment around these lines. can you please fix it?